kuma/contributions/jinja2/contributions/thank_you.html

```diff
@@ -5,46 +5,44 @@
-                            To build the commons, we depend on uncommon
-                            boosters like you. What you’ve given to MDN will
-                            stay here to build more of the tools and
-                            information we use every day. And it helps us
-                            expand MDN to devs everywhere.
-                        {% endtrans %}
-                    </p>
                     <p class="subtext">
                         {% trans %}
                             We are committed to providing the best source of
-                            free information about the internet to our
-                            community of devs around the world. Support us, and
-                            it will all come back around. It already has. And
-                            it will again.
+                            information about web standards. Your support enables
+                            MDN to build more of the content and tools you use every
+                            day, and helps us make sure MDN can be available to
+                            everyone, everywhere.
                         {% endtrans %}
                     </p>
                     <p class="subtext">{{_('Thank you again,')}}</p>
                     <p class="subtext">{{_('The MDN Team')}}</p>
 
                     <div class="button-tab-group">
-                        <a href="https://twitter.com/intent/tweet?text=I%27ve%20contributed%20to%20help%20make%20MDN%20sustainable,%20you%20can%20too%20here:%20developer.mozilla.com/en-US/contribute/" class="button" target="_blank" rel="noopener" aria-label="{{_('Share your support on Twitter')}}">{% include 'includes/icons/social/twitter.svg' %}</a>
+                        <a href="https://twitter.com/intent/tweet?text=I%20am%20a%20supporter%20of%20MDN,%20and%20you%20can%20too:%20https://developer.mozilla.com/en-US/contribute/" class="button" target="_blank" rel="noopener" aria-label="{{_('Share your support on Twitter')}}">{% include 'includes/icons/social/twitter.svg' %}</a>
                         <a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fdeveloper.mozilla.org%2Fen-US%2Fcontribute" class="button" target="_blank" rel="noopener" aria-label="{{_('Share your support on Facebook')}}">{% include 'includes/icons/social/facebook.svg' %}</a>
                     </div>
                 {% else %}
                     <h2 class="highlight highlight-spanned">
                         <span class="highlight-span">{{_('Ooops, something went wrong')}}</span>
-                        <p>
-                            <a href="{{ url('contribute') }}">{{_('Click here to go back')}}</a>
-                        </p>
                     </h2>
+                    <p>
+                        {% trans %}
+                        Please contact <a href="mailto:support@mozilla.com">support@mozilla.com</a>  for more information on this problem.
```

Comment:
Future task: We'll need to move the email outside of the translated string, and use a real email address (unless support@mozilla.com is the real email address).

Reply:
Hi John. Yep, I believe we're still waiting on the final support email address. As of yesterday, Kadir still hadn't received this so it might fall to you guys to update once you have it.

Reply:
The email address is mdn-support@mozilla.com